Source/i_oplmusic.c

```diff
@@ -51,7 +51,11 @@
 
 #define PERCUSSION_LOG_LEN 16
 
-typedef PACKED_STRUCT (
+#ifdef _MSC_VER
```

Comment:
Isn't this missing an additional check for ` _MSC_VER < 1800`? AFAIUI, `_MSC_VER >= 1800` does support the `__pragma()` syntax as defined in `doomtype.h`?

Reply:
All _MSC_VER support `#pragma` and _MSC_VER >= 1800 also support `__pragma`. I want to change this to a more readable version like this: https://github.com/spotify/linux/blob/master/drivers/staging/brcm80211/include/packed_section_start.h

Reply:
Alright, so could we mangle the _MSC_VER version handling into the logic in `doomtype.h`?

Reply:
New approach: https://github.com/fabiangreffrath/woof/pull/400/commits/4d79aeef8b14d65147f5a6bf97ba60ebfd52ba00